{"uid":"9adaa12088e377c3","name":"Авторизация пользователя с валидным логин и паролем через API","fullName":"tests.test_authorization.TestAuthUser#test_api_auth_valid_username_and_valid_password","historyId":"1396a5305a37c317f3d46e05e4e7128e","time":{"start":1738874917940,"stop":1738874918492,"duration":552},"description":"При проверке используются валидный логин и пароль пользователя Aleska","descriptionHtml":"<p>При проверке используются валидный логин и пароль пользователя Aleska</p>\n","status":"broken","statusMessage":"KeyError: 'access_token'","statusTrace":"self = <test_authorization.TestAuthUser object at 0x7fc253f0f920>\n\n    @allure.title('Авторизация пользователя с валидным логин и паролем через API')\n    @allure.description('При проверке используются валидный логин и пароль пользователя Aleska')\n    @allure.severity('Critical')\n    @pytest.mark.regression\n    def test_api_auth_valid_username_and_valid_password(self):\n        with allure.step('Открыть страницу логина'):\n>           self.api_authorization.user(self.User.LOGIN, self.User.PASSWORD)\n\ntests/test_authorization.py:147: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <model.api.authorization.ApiAuthorization object at 0x7fc252b0dc40>\nusername = 'Aleska', password = 'qwe123'\n\n    def user(self, username: str, password: str) -> None:\n        \"\"\"\n        Метод авторизации пользователя по средствам API.\n    \n        :param username: Логин пользователя\n        :param password: Пароль пользователя\n        \"\"\"\n        headers = {\n            'X-Client-Id': 'round',\n            'X-Client-Type': 'web',\n            'X-Language': 'ru'\n        }\n        data = {\n            'client_id': 'Krujok.Online.Mobile',\n            'grant_type': 'password',\n            'username': username,\n            'password': password\n        }\n    \n        response = requests.post(url=Api.TOKEN, headers=headers, data=data)\n>       access_token = response.json()[\"access_token\"]\nE       KeyError: 'access_token'\n\nmodel/api/authorization.py:30: KeyError","flaky":false,"newFailed":false,"newBroken":true,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_session_faker","time":{"start":1738874486972,"stop":1738874486974,"duration":2},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false},{"name":"driver","time":{"start":1738874917524,"stop":1738874917940,"duration":416},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false},{"name":"setup","time":{"start":1738874917940,"stop":1738874917940,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"При проверке используются валидный логин и пароль пользователя Aleska","status":"broken","statusMessage":"KeyError: 'access_token'","statusTrace":"self = <test_authorization.TestAuthUser object at 0x7fc253f0f920>\n\n    @allure.title('Авторизация пользователя с валидным логин и паролем через API')\n    @allure.description('При проверке используются валидный логин и пароль пользователя Aleska')\n    @allure.severity('Critical')\n    @pytest.mark.regression\n    def test_api_auth_valid_username_and_valid_password(self):\n        with allure.step('Открыть страницу логина'):\n>           self.api_authorization.user(self.User.LOGIN, self.User.PASSWORD)\n\ntests/test_authorization.py:147: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <model.api.authorization.ApiAuthorization object at 0x7fc252b0dc40>\nusername = 'Aleska', password = 'qwe123'\n\n    def user(self, username: str, password: str) -> None:\n        \"\"\"\n        Метод авторизации пользователя по средствам API.\n    \n        :param username: Логин пользователя\n        :param password: Пароль пользователя\n        \"\"\"\n        headers = {\n            'X-Client-Id': 'round',\n            'X-Client-Type': 'web',\n            'X-Language': 'ru'\n        }\n        data = {\n            'client_id': 'Krujok.Online.Mobile',\n            'grant_type': 'password',\n            'username': username,\n            'password': password\n        }\n    \n        response = requests.post(url=Api.TOKEN, headers=headers, data=data)\n>       access_token = response.json()[\"access_token\"]\nE       KeyError: 'access_token'\n\nmodel/api/authorization.py:30: KeyError","steps":[{"name":"Открыть страницу логина","time":{"start":1738874917941,"stop":1738874918491,"duration":550},"status":"broken","statusMessage":"KeyError: 'access_token'\n","statusTrace":"  File \"/usr/workspace/tests/test_authorization.py\", line 147, in test_api_auth_valid_username_and_valid_password\n    self.api_authorization.user(self.User.LOGIN, self.User.PASSWORD)\n  File \"/usr/workspace/model/api/authorization.py\", line 30, in user\n    access_token = response.json()[\"access_token\"]\n                   ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"attachmentsCount":0,"stepsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"attachmentsCount":0,"stepsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"driver::0","time":{"start":1738874918497,"stop":1738874918685,"duration":188},"status":"passed","steps":[],"attachments":[{"uid":"4fef06f4188d7cf4","name":"page_source","source":"4fef06f4188d7cf4.html","type":"text/html","size":39},{"uid":"72a3fdfe45b650d6","name":"screenshot","source":"72a3fdfe45b650d6.png","type":"image/png","size":10592},{"uid":"43fa67f3adabf459","name":"browser_logs","source":"43fa67f3adabf459.txt","type":"text/plain","size":0}],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":3,"stepsCount":0,"hasContent":true,"attachmentStep":false}],"labels":[{"name":"severity","value":"Critical"},{"name":"feature","value":"Авторизация пользователя"},{"name":"tag","value":"regression"},{"name":"tag","value":"authorization"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_authorization"},{"name":"subSuite","value":"TestAuthUser"},{"name":"host","value":"2fa5c50d49f6"},{"name":"thread","value":"1-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_authorization"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"critical","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":1,"broken":6,"skipped":0,"passed":54,"unknown":0,"total":61},"items":[{"uid":"4f75f53899e51223","status":"failed","statusDetails":"AssertionError: Ожидалась страница https://staging.round.zone/rubric, но открылась https://staging.round.zone/. Страница не загрузилась за 10 секунд.","time":{"start":1738871535420,"stop":1738871572989,"duration":37569}},{"uid":"533faf4936596b25","status":"passed","time":{"start":1738864905043,"stop":1738864909830,"duration":4787}},{"uid":"d1210528fc8abb87","status":"passed","time":{"start":1738268556054,"stop":1738268563498,"duration":7444}},{"uid":"d5c05ab770a45be7","status":"passed","time":{"start":1738098995650,"stop":1738099002398,"duration":6748}},{"uid":"6832413afebd900d","status":"passed","time":{"start":1738005006944,"stop":1738005014775,"duration":7831}},{"uid":"cab141389226d882","status":"passed","time":{"start":1738002338386,"stop":1738002343888,"duration":5502}},{"uid":"49d98fd51dc6f54d","status":"passed","time":{"start":1737927950950,"stop":1737928014729,"duration":63779}},{"uid":"6ef344d5e3d72864","status":"passed","time":{"start":1737842612840,"stop":1737842622282,"duration":9442}},{"uid":"ef533e4ad6464f74","status":"passed","time":{"start":1737831903986,"stop":1737831910555,"duration":6569}},{"uid":"daeec1135f684bb","status":"passed","time":{"start":1737799238921,"stop":1737799245727,"duration":6806}},{"uid":"f38f654567bd9923","status":"passed","time":{"start":1737757514460,"stop":1737757519645,"duration":5185}},{"uid":"3aaa4fee4357e6b7","status":"passed","time":{"start":1737744472310,"stop":1737744477434,"duration":5124}},{"uid":"77dc845f6087a0e5","status":"passed","time":{"start":1737652253427,"stop":1737652260995,"duration":7568}},{"uid":"769fcedd8170f81c","status":"passed","time":{"start":1737569875159,"stop":1737569883062,"duration":7903}},{"uid":"4b730976def764fc","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/chromium-browser is no longer running, so ChromeDriver is assuming that Chrome has crashed.)","time":{"start":1737568960991,"stop":1737568960991,"duration":0}},{"uid":"49b91a53ca6d85dd","status":"passed","time":{"start":1734034286316,"stop":1734034291636,"duration":5320}},{"uid":"af5f1a9ca18d5673","status":"broken","statusDetails":"selenium.common.exceptions.TimeoutException: Message:","time":{"start":1733936961309,"stop":1733936996315,"duration":35006}},{"uid":"24167f51472f75a1","status":"passed","time":{"start":1733865448920,"stop":1733865454455,"duration":5535}},{"uid":"955ac0d377cf726c","status":"broken","statusDetails":"selenium.common.exceptions.TimeoutException: Message:","time":{"start":1733859897987,"stop":1733859933844,"duration":35857}},{"uid":"a10777b89236b2ac","status":"passed","time":{"start":1733856557604,"stop":1733856563152,"duration":5548}}]},"tags":["authorization","regression"]},"source":"9adaa12088e377c3.json","parameterValues":[]}